quiz.rb

```diff
@@ -2,19 +2,95 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+
+class Dog
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+  attr_reader :name, :breed, :age
+end
```

Comment:
Nice! One small point - the convention is to put `attr_accessor` and `attr_reader` before the `initialize` method.